jenny/values.yaml

```diff
@@ -0,0 +1,92 @@
+# Jenny Chart default values
+
+nameOverride: ""
+fullnameOverride: ""
+
+image:
+  repository: 49705lm8.c1.gra9.container-registry.ovh.net/jenny/jenny
+  pullPolicy: Always
+  tag: "production-latest"
+  dockerconfigjson: ""  # cf https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
```

Comment:
Are you sure it's required if you point to an existing image pull Secret?

Reply:
are you saying that this conf should live outside of this chart ?

Reply:
Yup. I think this secret generation should be prepared at a cluster / namespace level. Current chart should not be responsible for this.